controller/workitem_blackbox_test.go

```diff
@@ -1973,7 +1973,7 @@ func (s *WorkItem2Suite) TestWI2FailOnDelete() {
 
 func (s *WorkItem2Suite) TestWI2CreateWithArea() {
 	// given
-	_, areaInstance := createSpaceAndArea(s.T(), gormapplication.NewGormDB(s.DB))
+	areaInstance := tf.NewTestFixture(s.T(), s.DB, tf.Areas(1)).Areas[0]
```

Comment:
Like I mentioned above, I think that the `createSpaceAndArea` function does more than just creating an area.

Reply:
Please see https://github.com/fabric8-services/fabric8-wit/pull/1941/commits/824592e1691a52ee4d2be7cd780f9bcd84222029